code/Controllers/CMSPagesController.php

```diff
@@ -31,35 +31,4 @@ public function isCurrentPage(DataObject $record)
     {
         return false;
     }
-
-    public function Breadcrumbs($unlinked = false)
```

Comment:
This bit of logic was the reason why we were ending up with duplicate crumbs. Basically, the `CMSMain` would want to build the crumb once and this would build it again. 

I really don't see why this was put on a separate controller. Having all the Breadcrumb logic on CMSMain simplifies everything.